features/step_definitions/sign_up.rb

```diff
@@ -55,7 +57,7 @@
 
 # Unsuccessful registration: invalid email/password
 
-When('I fill the {string} field with invalid information like: {string}') do |field, information|
+And('I fill the {string} field with invalid information like: {string}') do |field, information|
   enter_text(:id, information, field)
```

Comment:
when you are going to perform an action it is good practice to validate that if it is visible and also if it is enabled, this in the case of buttons and fields